roles/ceph-mon/tasks/docker/pre_requisite.yml

```diff
@@ -15,14 +15,33 @@
     state: present
   when: ansible_os_family == 'Debian'
 
+- name: install epel-release on redhat
+  yum:
+    name: epel-release
```

Comment:
`epel-release` turns out to be in centos yum repo, so we can just yum install it.